### PR TITLE
artemis: we test also error messages

### PR DESCRIPTION
**Pull request by kadhikari**
6 tests on error added for bibus.

Concerned Ticket: http://jira.canaltp.fr/browse/ITI-343

Attention: Only those 6 tests added will fail but not others

### CODE REVIEW

File: artemis/default_checker.py [hunk 1]
```diff
@@ -28,8 +28,13 @@
     'type': fields.Raw,
 }
 
+no_solution = {
```
Comment (antoine-de) [inline artemis/default_checker.py]: it isn't a `no_solution` it' an `error` no ?

File: artemis/tests/bibus_test.py [hunk 2]
```diff
@@ -67,6 +67,39 @@ def test_bibus_11(self):
...
+                     datetime_represents="arrival", walking_speed="0.83", max_duration_to_pt="1200")
+
+    def test_no_origin(self):
+        self.journey(_from="poi:222500305513897_7",
+                     to="stop_area:BIB:SA:1210", datetime="20041214T220000",
+                     datetime_represents="departure", walking_speed="0.83", max_duration_to_pt="50")
+
+    def test_no_destination(self):
+        self.journey(_from="stop_area:BIB:SA:9",
+                     to="poi:222500305513897_7", datetime="20041214T220000",
```
Comment (antoine-de) [inline artemis/tests/bibus_test.py]: even if that quite clear, you can make this clearer with a more concrete to id like `poi:unexisting_poi`

File: artemis/tests/bibus_test.py [hunk 3]
```diff
@@ -67,6 +67,39 @@ def test_bibus_11(self):
...
+    '''
+    Tests of responses with error message.
+    '''
+    def test_date_out_of_bound(self):
+        self.journey(_from="stop_area:BIB:SA:527",
+                     to="stop_area:BIB:SA:9", datetime="20051214T070000",
+                     datetime_represents="arrival", walking_speed="0.83", max_duration_to_pt="1200")
+
+    def test_no_origin(self):
+        self.journey(_from="poi:222500305513897_7",
```
Comment (antoine-de) [inline artemis/tests/bibus_test.py]: same with clearer id

File: artemis/tests/bibus_test.py [hunk 4]
```diff
@@ -67,6 +67,41 @@ def test_bibus_11(self):
...
+                     to="stop_area:BIB:SA:1210", datetime="20041214T220000",
+                     datetime_represents="departure", walking_speed="0.83", max_duration_to_pt="50")
+
+    def test_no_destination(self):
+        self.journey(_from="stop_area:BIB:SA:9",
+                     to="poi:222500305513897_7", datetime="20041214T220000",
+                     datetime_represents="departure", walking_speed="0.83", max_duration_to_pt="50")
+
+    def test_no_solution(self):
```
Comment (antoine-de) [inline artemis/tests/bibus_test.py]: can you add a comment explaining why there is no solution ?

File: artemis/tests/bibus_test.py [hunk 5]
```diff
@@ -67,6 +67,41 @@ def test_bibus_11(self):
...
+        self.journey(_from="stop_area:BIB:SA:9",
+                     to="poi:222500305513897_7", datetime="20041214T000000",
+                     datetime_represents="departure", walking_speed="0.83", max_duration_to_pt="1200")
+
+    @xfail(reason="This test needs a correction in experimental", raises=AssertionError)
+    def test_from_unknown_object(self):
+        self.journey(_from="stop_area:BIB:SA:",
```
Comment (antoine-de) [inline artemis/tests/bibus_test.py]: what object is unknown ?

File: artemis/tests/bibus_test.py [hunk 6]
```diff
@@ -67,6 +67,41 @@ def test_bibus_11(self):
...
+                     to="stop_area:BIB:SA:9", datetime="20041214T000000",
+                     datetime_represents="departure", walking_speed="0.83", max_duration_to_pt="1200")
+
+    @xfail(reason="This test needs a correction in experimental", raises=AssertionError)
+    def test_to_unknown_object(self):
```
Comment (antoine-de) [inline artemis/tests/bibus_test.py]: same here

File: artemis/default_checker.py (hunk 1 above)
Comment (kadhikari) [inline artemis/default_checker.py]: You are right

File: artemis/tests/bibus_test.py (hunk 3 above)
Comment (kadhikari) [inline artemis/tests/bibus_test.py]: Here, no_origin is not the result of not existing poi but due to the parameter  "&max_duration_to_pt=50" from origin

File: artemis/tests/bibus_test.py (hunk 2 above)
Comment (kadhikari) [inline artemis/tests/bibus_test.py]: Here, no_destination is not the result of not existing poi but due to the parameter  "&max_duration_to_pt=50" from destination

File: artemis/tests/bibus_test.py (hunk 4 above)
Comment (kadhikari) [inline artemis/tests/bibus_test.py]: ok

File: artemis/tests/bibus_test.py (hunk 3 above)
Comment (antoine-de) [inline artemis/tests/bibus_test.py]: ah ok, you should thus add a comment

**Pull request merged**